platforms/pomelo/src/main.rs

```diff
@@ -239,8 +239,6 @@ async fn kernel_entry() {
             let dt = now.signed_duration_since(then).to_std().unwrap();
             trace!("timer: slept for {dt:?}");
             kernel.timer().force_advance(dt);
-        } else {
-            warn!("timer worm");
```

Comment:
i kinda liked the worm but i agree that we should probably not log this at WORM...er, WARN...

Reply:
agree - I just wanted it to visually stick out. May it rest in peace!